core/frontend/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue';
+import App from './App.vue';
+import router from './router';
+import store from './store';
+import vuetify from './plugins/vuetify';
+
+new Vue({
+  router,
+  store,
+  vuetify,
+  render: (h) => h(App),
+}).$mount('#app');
```

Comment:
You are missing the service worker registration for PWA:
```
if("serviceWorker" in navigator) {
    navigator.serviceWorker
        .register("service-worker.js")
        .then(function() { console.log("Service Worker Registered.") })
}
```

`service-worker.js` should be generated by the PWA system that you are using here (check your package.json and vue.config.js)

Reply:
It's not needed. Tested here and it's working without it. It's generated by vue.